### ISSUE TITLE
Usage with other wrappers

### ISSUE THREAD

**Issue by andersstorhaug** [briankendall/devreorder#36]
I'm using this wrapper in an arcade cabinet where I also use [Boomslangnz/FFBArcadePlugin](FFBArcadePlugin) to add/improve force-feedback for driving games in MAME and M2Emulator.

The important part though is that FFBArcadePlugin also wraps `dinput8.dll` in some cases (when used with M2Emulator for instance).
I'm guessing that a system-wide install in this scenario would work, but, I'm wondering: is this scenario somehow supported for a local install?

I'm hoping that the following may possibly work __in a local directory__:
- `.\dinput8.dll` from devreorder
- `.\dinput8org.dll` from FFBArcadePlugin

I'll give this a shot when I have the time. If it works correctly, maybe this type of install is worth mentioning in `README.md`?

And, as an aside I prefer a local install because I use/maintain a setup script that configure all of my arcade emulators. Keeping everything localized is easier to manage for me.

**Comment by andersstorhaug** [comment]
Poking around in sources I see `ChainLoadFileName` (awesome). Maybe that's all that I need here?

**Comment by briankendall** [comment]
Yes `ChainLoadFileName` should do the trick. I completely forgot that's in there, probably because I copied it out of x360ce! I'll be curious to hear whether it works.
